app/feed/tests/test_process_alignment_feed.py

```diff
@@ -0,0 +1,53 @@
+from app.network_x_tools.network_x_utils import network_x_utils
+
+from app.feed.process_alignment_feed import (
+    find_alignment_solution_iris,
+    get_solution_nodes,
+    CONVERSATION_SOLUTION_NAME,
+    POPULAR_SOLUTION_NAMES,
+    POPULAR_SOLUTION_COUNT,
+    UNPOPULAR_SOLUTION_COUNT,
+)
+
+
+def test_find_alignment_solution_iris():
+    nx = network_x_utils()
+    solution_nodes = get_solution_nodes()
+    iri_name_map = {}
+
+    for node in solution_nodes:
+        nx.set_current_node(node)
+        iri_name_map[nx.get_node_id()] = node["label"]
+
+    result_solution_iris = find_alignment_solution_iris()
+    conversation_solution_count = 1
+    total_solutions_count = (
+        POPULAR_SOLUTION_COUNT + UNPOPULAR_SOLUTION_COUNT + conversation_solution_count
+    )
+    assert (
+        len(result_solution_iris) == total_solutions_count
+    ), f"Total solutions cound should be {total_solutions_count}"
```

Comment:
Don't move all asserts to the bottom, try to keep them near variables you use per assert